### ISSUE TITLE
Cannot see performance data in sentry

### ISSUE THREAD

**Issue by Linspirit98** [getsentry/sentry-dart#1245]
### Description

<img width="744" alt="image" src="https://user-images.githubusercontent.com/51519489/214541087-962a93d8-1318-4e1c-9aac-2015af6927e3.png">

I added Sentry in my project(as shown in the image above). I created an intentional error and sentry caught the exception successfully. I can see assertion error in Issues section. However, I cannot seem to see any data in Performance section. I thought there's automatic transaction catch. 

I also tried implementing sentry_dio and added catchTransaction method, but it's also not showing anything in my performance. 

Hope someone can help! Thanks in advance.

flutter: v3.3.0
dart: 2.18.0
sentry_flutter: 6.18.2
sentry_dio: 6.18.3

**Comment by marandaneto** [comment]
Take a look at the [automatic instrumentation](https://docs.sentry.io/platforms/flutter/performance/instrumentation/automatic-instrumentation/) section of the docs.
Either you have to [manually start transactions](https://docs.sentry.io/platforms/flutter/performance/instrumentation/custom-instrumentation/) or set up the integrations that automatically do it.

**Comment by Linspirit98** [comment]
> Take a look at the [automatic instrumentation](https://docs.sentry.io/platforms/flutter/performance/instrumentation/automatic-instrumentation/) section of the docs. Either you have to [manually start transactions](https://docs.sentry.io/platforms/flutter/performance/instrumentation/custom-instrumentation/) or set up the integrations that automatically do it.

I've tried debug mode and I saw the values of traceSampleRate and DSN showing up correctly. However, I see this messaage
```
Not going to enable SentryPerformanceTrackingIntegration because isTracingEnabled is disabled.
```
in my debug console. traceSampleRate is assigned so I don't understand why this message shows up. There are no data showing up in performance. I tried to implement manually but it's also not getting anywhere. Do you know what I'm possibly doing wrong?

**Comment by marandaneto** [comment]
Not sure, can you provide a minimal reprocible exampe?
See docs, https://docs.sentry.io/platforms/flutter/
The Flutter SDK already runs the App in a ZonedGuard, so your wrapper `runZoned` isn't needed, could be related to that?

**Comment by Linspirit98** [comment]
> Not sure, can you provide a minimal reprocible exampe? See docs, https://docs.sentry.io/platforms/flutter/ The Flutter SDK already runs the App in a ZonedGuard, so your wrapper `runZoned` isn't needed, could be related to that?

I removed zonedGuard but still nothing in performance. I tried https://github.com/getsentry/sentry-dart example project. I assume it should automatically start collecting performance data (such as cold start warm start) right? but there's nothing. In the example project, the only transaction data I can see is dart_try_catch and all the other ones are not shown or counted toward performance. I'm also confused why transaction doesn't show up when I made a dio request in the example.

Also, in the example project, I also see this message. Is this normal?
```
Not going to enable SentryPerformanceTrackingIntegration because isTracingEnabled is disabled.
